test/image/mocks/bar-like_textangle45.json

```diff
@@ -144,7 +144,7 @@
         "legend": {
             "orientation": "h",
             "title": {
-                "text": "<b>bar, funnel & waterfall</b><br>textangle:-45"
```

Comment:
What's the purpose of removing all the `<b>` tags in titles?

Reply:
Good question.
To make it easier to see what the default look like.

Reply:
OK - we have other examples of styled text, so this is fine.